src/client/Client.cc

```diff
@@ -6011,6 +6013,16 @@ void Client::tick()
     }
   }
 
+  {
```

Comment:
After using right data types, this block looked overlong and unreadable. So, I've declared a couple of variables to express the logic succinctly and, then, added a block-scope since these variables won't be required for long.

Reply:
sure